_data-prepper/peer-forwarder.md

```diff
@@ -0,0 +1,183 @@
+---
+layout: default
+title: Peer Forwarder
+nav_order: 12
+---
+
+# Peer Forwarder
+
+Peer Forwarder is an HTTP service which performs peer forwarding of an `event` between Data Prepper nodes for aggregation. Currently, Peer Forwarder is supported by `aggregate`, `service_map_stateful`, `otel_trace_raw` processors.
+
+Peer Forwarder groups events based on the identification keys provided by the processors. For `service_map_stateful` and `otel_trace_raw`, the identification key is `traceId` by default and cannot be configured. The value is configurable for the `aggregate` processor using the `identification_keys` configuration option. You can find more information about [identification keys](https://github.com/opensearch-project/data-prepper/tree/main/data-prepper-plugins/aggregate-processor#identification_keys) on the aggregate processor page.
+
+
+Peer discovery is currently provided by either a static list or by a DNS record lookup or AWS Cloudmap.  
+
+## Discovery modes
+
+See the following information about discovery modes below.
+
+### Static
+
+Static discover mode allows a Data Prepper node to discover nodes using a list of IP addresses or domain names.
+```yaml
+peer_forwarder:4
+  discovery_mode: static
+  static_endpoints: ["data-prepper1", "data-prepper2"]
+```
+
+### DNS lookup
+We recommend using DNS discovery over static discovery when scaling out a Data Prepper cluster. The core concept is to configure a DNS provider to return a list of Data Prepper hosts when given a single domain name. This is a [DNS A Record](https://www.cloudflare.com/learning/dns/dns-records/dns-a-record/) which indicates a list of IP addresses of a given domain.
```

Comment:
Should we avoid using the verbiage "We recommend" since this is an open source project? Suggestion: "DNS discovery is preferred over static..."

Add a comma after DNS A Record.